stable/traefik/templates/default-cert-secret.yaml

```diff
@@ -10,6 +10,14 @@ metadata:
     heritage: {{ .Release.Service | quote }}
 type: Opaque
 data:
+{{- if .Values.ssl.generateTLS }}
+  {{- $ca := genCA "default-ca" 365 }}
+  {{- $cn := .Values.ssl.defaultCN }}
```

Comment:
Default CN should be optional.

Reply:
All values but `generateTLS` should be optional (aka need a test for existence before use)